### ISSUE TITLE
Call to selenium server by 127.0.0.1

### ISSUE THREAD

**Issue by alexdashkov** [Volentix/venue-server#140]
I see some weird thing, that you call a selenium hub server using `127.0.0.1`. 
Can you confirm me that this part is working at the cluster? 

Found at: https://github.com/Volentix/venue-server/blob/master/venue/scrapers/bitcointalk.py#L63

**UPD:**
Unfortunately, I can't reproduce a normal call of this code part (I didn't find a good user), but you can try to run only this snippet in the Docker container: 

```
$ docker-compose run --rm web python manage.py shell

from selenium import webdriver
from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
driver = webdriver.Remote(
                'http://127.0.0.1:4444/wd/hub',
                DesiredCapabilities.CHROME
)
```

Connects to #187

**Comment by shawnlauzon** [comment]
@alexdashkov I believe this can be resolved with #187

**Comment by alexdashkov** [comment]
Yes, it is.
